### PR TITLE
Add my apprenticeship post

### PR DESCRIPTION
**Pull request by SengitU**
Changes proposed in the PR fix/change the following:
- Add my apprenticeship post

To the reviewer: *If you approve this PR, please merge it right away too.*
(Why merging it right away? It makes the PR valuable as soon as it is approved.
The author wrote it with the intention for it to go live, so let's do them the favor as soon as possible :).
Thanks.)

### CODE REVIEW

File: _posts/2018-07-27-my-apprenticeship.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+---
+
+My days as an apprentice are slowly coming to an end. As the final day arrives, I have an irresistible desire to share. I would like to share how I felt, how I learnt, how I've been changed during my apprenticeship.
+
+It's really hard to structure everything flowing in my mind, so I'll follow the three phases I had during the apprenticeship.
+
+## The Basics
+
+Contrary to its name, this phase was the hardest part of the apprenticeship for me. During this phase, you need to learn important techniques about not just software development, but also human skills like communication and facilitation. Apart from the learnings, you need to set your mind to learn. It's easy to learn more about what you are currently doing, but by doing so you channel yourself more on the way you are already doing things. As you specialize more on a technique, you lost your flexibility and you create your own bubble. **The very first step I took was to wear the white belt**.
+
```
Comment (Arturszott) [inline _posts/2018-07-27-my-apprenticeship.md]: Image of white belt here!

Comment (Arturszott) [reply _posts/2018-07-27-my-apprenticeship.md]: There are not many good images of white belt, but your road started there so maybe something like this?

File: _posts/2018-07-27-my-apprenticeship.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+### Wearing the White Belt
+
+I wanted to keep my flexibility while keep learning during my apprenticeship. But as I mentioned, it's not possible to see different opinions if you learn more about what you are already doing. After years of hands-on professional experience and much more social experience with the help of the [IEEE](https://www.ieee.org/), I almost always had an idea about the topic. To wear the white belt, you need to silence that inner voice. You need to talk less and observe more just to find out the different point of views. It's not as easy as sounds, during my life I saw so many people just can't observe. Knowing that every mind is unique helped me to observe. When you start to observe, you open yourself to the possibilities, new ideas and new techniques.
```
Comment (Arturszott) [inline _posts/2018-07-27-my-apprenticeship.md]: `After years of hands-on professional experience and much more social experience with the help of the [IEEE](https://www.ieee.org/), I almost always had an idea about the topic. ` About what topic?

Comment (Arturszott) [inline _posts/2018-07-27-my-apprenticeship.md]: ...It's not as easy as `it` sounds...

File: _posts/2018-07-27-my-apprenticeship.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+
+When I started to observe, finally I wore the white belt. I figured out what needs to be learnt and I started to bake my knowledge with my new learnings.
```
Comment (Arturszott) [inline _posts/2018-07-27-my-apprenticeship.md]: I would avoid word `bake` when you write about white belt. Just `expand` will be ok

File: _posts/2018-07-27-my-apprenticeship.md [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+
+### Self-Discipline
+
+You should've been there to see my glowing eyes when I opened myself to learning. The excitement I felt after I've decided what to learn during the 6 months... I want everyone to be an apprentice one day just to feel that way. Unfortunately, excitement alone is not enough. First of all, you need to come up with a plan, but having a solid plan is still not enough. You must have the discipline to follow the plan, otherwise, you can easily waste your apprenticeship without noticing. During the apprenticeship, you learn more about yourself, and I've learnt how much I like to stick to a plan. With the help of my colleagues, I came up with a great plan that helped me during these 6 months.
```
Comment (Arturszott) [inline _posts/2018-07-27-my-apprenticeship.md]: `You should've seen my glowing eyes when`

**Comment by SengitU** [comment]
@Arturszott thanks for your notes, I've updated my post according to them. Only problem is, I couldn't find any good white belt picture.

**Pull request merged**